setup.py

```diff
@@ -42,7 +42,7 @@ def run_tests(self):
       url="http://github.com/confluentinc/ducktape",
       packages=find_packages(),
       package_data={'ducktape': ['templates/report/*']},
-      install_requires=['jinja2', 'requests', 'pysistence'],
+      install_requires=['jinja2', 'requests', 'pysistence', 'zmq'],
```

Comment:
The `zmq` package seems to suggest you should be using `pyzmq`: https://pypi.python.org/pypi/zmq/0.0.0